Scripts/NuGet/EdFi.Installer.SandboxAdmin/Install-EdFiOdsSandboxAdmin.psm1

```diff
@@ -198,24 +198,18 @@ function Get-SandboxAdminPackage {
         $Config.WebConfigLocation = Resolve-Path (Join-Path $packageDir "Web.config")
```

Comment:
I was able to run the test.ps1 by following these steps:
- run `build-package.ps1`
- copying `Ed-Fi-ODS-Implementation\logistics\scripts\modules\utility\hashtable.psm1` to `Ed-Fi-ODS-Implementation\Scripts\NuGet\EdFi.Installer.SandboxAdmin\Ed-Fi-ODS-Implementation\logistics\scripts\modules\utility\hashtable.psm1` (we'll fix this manual step with another ticket)
- updating the `PackageVersion` to `5.1.0-b12760` here: https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/blob/1e8ec4aeac24cf8b3d3883394cea85eaa99dc118/Scripts/NuGet/EdFi.Installer.SandboxAdmin/test.ps1#L26
- run `test.ps1 CustomPackage`

- you should then see the error `Cannot find path 'C:\edfi\Ed-Fi-ODS-Implementation\Scripts\NuGet\EdFi.Installer.SandboxAdmin\downloads\EdFi.Suite3.Ods.SandboxAdmin.5.1.0-b12760\Web.config\appsettings.json' because it does notexist.
`

let's update this variable and it's name and make sure the rest of this test.ps1 will run.